lib/ex_machina.ex

```diff
@@ -49,6 +49,33 @@ defmodule ExMachina do
       def build_list(number_of_factories, factory_name, attrs \\ %{}) do
         ExMachina.build_list(__MODULE__, number_of_factories, factory_name, attrs)
       end
+
+      def create(_) do
+        raise_error_helper("create/1", "insert/1")
+      end
+
+      def create(_, _) do
+        raise_error_helper("create/2", "insert/2")
+      end
+
+      def create_pair(_, _) do
+        raise_error_helper("create_pair/2", "insert_pair/2")
+      end
+
+      def create_list(_, _, _) do
+        raise_error_helper("create_list/3", "insert_list/3")
+      end
+
+      defp raise_error_helper(old_function, new_function) do
```

Comment:
Yeah that sounds good!
> On May 11, 2016, at 10:14 AM, David Kuhta <notifications@github.com> wrote:
> 
> In lib/ex_machina.ex <https://github.com/thoughtbot/ex_machina/pull/129#discussion_r62855385>:
> 
> > +        raise_error_helper("create/1", "insert/1")
> > +      end
> > +
> > +      def create(_, _) do
> > +        raise_error_helper("create/2", "insert/2")
> > +      end
> > +
> > +      def create_pair(_, _) do
> > +        raise_error_helper("create_pair/2", "insert_pair/2")
> > +      end
> > +
> > +      def create_list(_, _, _) do
> > +        raise_error_helper("create_list/3", "insert_list/3")
> > +      end
> > +
> > +      defp raise_error_helper(old_function, new_function) do
> In favor as well, but how about raise_function_replaced_error. Given the old_function & new_function params?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/ex_machina/pull/129/files/8ee1e519cc97978edb17218bc2a7eb75046dff0d#r62855385>